app.js

```diff
@@ -0,0 +1,24 @@
+$( document ).ready(function(event) {
+
+	$('.readmore').click(function(event){
```

Comment:
You have two ```.readmore``` on the page, you need to separate the click handlers otherwise, they will activate the same readmore sections.